README.md

```diff
@@ -108,7 +108,7 @@ Project section was buiilt with HTML and CSS
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://bellagirl-maker.github.io/Portfolio/)
```

Comment:
- [ ] You have added the link in the README. However, you need to ensure that the link is visible in the README file of the PR description ☺️ .

Reply:
@AlphaNtihinduka The link is now visible in the README file of the PR description.